.github/workflows/deploy-to-production-cdn.yaml

```diff
@@ -0,0 +1,20 @@
+name: Deploy files to staging cdn
+
+on:
+  push:
+    branches:
+    - master
```

Comment:
Should we rename to branch to main before going to prod? Or is the timing for doing that irrelevant?

Reply:
I think that is irrelevant, small change in the pipeline. There are other external pipelines that push to master so we need to sync that change with them